src/app/TideMatchSet.h

```diff
@@ -103,6 +111,48 @@ class TideMatchSet {
     bool sp
   );
 
+  /// added by Yang
+  static void writeHeadersDIA(ofstream* file, bool sp);
```

Comment:
Why is line 115 and 116 split into 2?
Usually in Crux we combine the two lines together.